lib/srv/app/server_test.go

```diff
@@ -397,21 +398,25 @@ func TestAuthorizeWithLocks(t *testing.T) {
 // TestGetConfigForClient verifies that only the CAs of the requested cluster are returned.
 func TestGetConfigForClient(t *testing.T) {
 	// TODO(r0mant): Implement this.
+	t.Skip("Not Implemented")
```

Comment:
I wonder if it's even worth keeping these around. Ideally, IMO, we should get automated coverage reports, so we can see how each package stands - this should make it clear which critical packages are lacking in the test department.

Reply:
That's definitely on my list (it's a _long_ list...). But, right now, at the very least they should not be reporting a pass.

Reply:
That's fair, thanks.